.github/workflows/buildpush.yaml

```diff
@@ -0,0 +1,78 @@
+---
+name: Build and Push Images
```

Comment:
Build and push receipe containers?

Reply:
I thought this was implied by the repo we are in.

Reply:
Sure we can remove "recipe" if you want.